.env.example

```diff
@@ -0,0 +1,38 @@
+# Get key from https://platform.openai.com/account/api-keys
+# This gives the user access to the OpenAI API, which is necessary for the Agent to function. 
+OPENAI_API_KEY=<YOUR_API_KEY>
+
+# For a structure running in Griptape Cloud, set this variable to: https://cloud.griptape.ai
+# If running against Skatepark, set this environment variable to tell the client which URL Skatepark is listening on.
+# Default is https://127.0.0.1:5000, but is configurable when running gt skatepark start. 
+GT_CLOUD_BASE_URL=
+
+# The Skatepark emulator does not require Griptape Cloud API Key.  
+# Structures in Griptape Cloud require a valid API key to authorize your client to call them. 
+# Generate a key from https://cloud.griptape.ai/account/api-keys 
+GT_CLOUD_API_KEY=<YOUR_API>
+
+# Structure ID from the Griptape Cloud or Skatepark
+# For Griptape Cloud: get from https://cloud.griptape.ai/structures 
+# For Skatepark: 
+# Once skatepark is started with 'gt skatepark start' 
+# get the id from running 'gt skatepark register --main-file <path-to-main-file>'
```

Comment:
Can we populate the path-to-main-file for this repo?

Reply:
This repo doesn't have a 'structure' in it - it has a structure being built in chat_local but doesn't have an isolated structure file with a structure.py / requirements.txt . Skatepark has to be ran / the structure has to be registered in a different repo or has to be a file of their own creation (if they make one in this repository). I'll try to make that more clear in the README / .env